libr/util/unum.c

```diff
@@ -73,7 +76,7 @@ R_API RNum *r_num_new(RNumCallback cb, RNumCallback2 cb2, void *ptr) {
 }
 
 R_API void r_num_free(RNum *num) {
-	R_FREE (num);
+	r_unref (num, free);
```

Comment:
use r_num_unref (num); instead. the r_num_free callback is the one that will be used and should be used

Reply:
ok 908b0e042e935b2324aa699c6229adf83d47b5bd